robottelo/cli/factory.py

```diff
@@ -1773,13 +1773,6 @@ def _setup_org_for_a_rh_repo(options=None):
         env_id = make_lifecycle_environment({'organization-id': org_id})['id']
     else:
         env_id = options['lifecycle-environment-id']
-    # Clone manifest and upload it
-    with manifests.clone() as manifest:
-        ssh.get_client().put(manifest, manifest.filename)
-    try:
-        Subscription.upload({'file': manifest.filename, 'organization-id': org_id})
-    except CLIReturnCodeError as err:
-        raise CLIFactoryError(f'Failed to upload manifest\n{err.msg}')
     # Enable repo from Repository Set
     try:
```

Comment:
Don't we need a manifest in order to enable the repo and sync below?

Reply:
Also can you remove 
```
2. Clones and uploads manifest.
```
from the docstring steps.

Reply:
Yes, we do need a manifest to enable and sync the repo. This change assumes that manifests will be provided by manifester fixtures, and I've updated the existing tests that use `setup_org_for_a_rh_repo` with `force_use_cdn=True` to reflect that assumption. An option to preserve this functionality would be to import Manifester to cli/factory.py and call it directly at this point, but my preference would be to standardize on using manifester fixtures for tests' manifests, rather than having manifests be provided by various helper functions.

And sure, I'll update the docstring.